docs/Configuring-Orion/TLS.md

```diff
@@ -3,15 +3,162 @@ description: TLS
 
 # TLS
 
-Orion supports the TLS (Transport Layer Security) protocol to enable secure communications.
+Orion supports the Transport Layer Security (TLS) protocol to enable secure communications. TLS is used between Orion nodes only. It is not used between Pantheon and Orion nodes.
```

Comment:
It is not used -> TLS is not used 

Suggested change because replacing _It_ with what's it's referrring to is easier for English as a second language users and better for translation if we end up doing this in the future.